ldelf/ta_elf.c

```diff
@@ -171,6 +192,20 @@ static void save_hashtab_from_segment(struct ta_elf *elf, unsigned int type,
 	}
 }
 
+static bool check_addr_size(struct ta_elf *elf, void *addr, size_t sz)
```

Comment:
This is not used in this commit. Wrong rebase?

Reply:
Exactly how this ended up here I don't know. I'm removing it and will force push since it messes with the diff in "ldelf: check that hashtab is in range before use".